bin/postgres-ha/bootstrap/create-from-existing.sh

```diff
@@ -24,4 +24,7 @@ echo_info "Bootstrapping a new PostgreSQL cluster using an existing PGDATA direc
 mv "${PATRONI_POSTGRESQL_DATA_DIR}_tmp" "${PATRONI_POSTGRESQL_DATA_DIR}"
 err_check "$?" "Initialize Existing PGDATA" "Could not initialize cluster using existing PGDATA directory"
 
+# ensure the PGDATA directory has the proper permissions
+chmod 0700 "${PATRONI_POSTGRESQL_DATA_DIR}"
```

Comment:
```suggestion
chmod u+rwx,go-rwx "${PATRONI_POSTGRESQL_DATA_DIR}"
```

Reply:
Was essentially just staying consistent with our existing scripts:

https://github.com/CrunchyData/crunchy-containers/blob/742926001ed1983c39f683b43b9eacb3b4419ef9/bin/postgres-ha/bootstrap/pre-bootstrap.sh#L469

Reply:
That's fine, but recommending better(?) syntax?

I typically use the numbers on my own.

Reply:
Cool, definitely not a problem updating.  The commit has updated per the syntax recommended above.